packages/shared/src/components/cards/AdLink.tsx

```diff
@@ -11,10 +11,10 @@ export type AdLinkProps = {
 export default function AdLink({ ad, onLinkClick }: AdLinkProps): ReactElement {
   return (
     <CardLink
-      href={ad.link}
+      href={ad?.link}
```

Comment:
Shouldn't they always have a link and description?

Reply:
Yes, unintended